articles/machine-learning/how-to-manage-quotas.md

```diff
@@ -112,10 +113,15 @@ Azure Machine Learning managed online endpoints have the following limits.
 | Number of instances per deployment | 20 |
 | Max payload size at endpoint level |1.5 MB |
 | Max request time out at endpoint level  | 60 seconds |
-| Total QPS at endpoint level for all deployments  | 100 |
-
-<sup>1</sup> Single dashes like, `my-endpoint-name`, are accepted in endpoint and deployment names
+| Total requests per second at endpoint level for all deployments  | 100 <sup>2</sup> |
```

Comment:
Do we allow change of the limits for all of them? I recall connection number was static

Reply:
yes, we allow changing all: https://msdata.visualstudio.com/Vienna/_git/inference-deployment-plane?path=/deploy/specs/prod/prod-base.yaml&version=GBmain&line=79&lineEnd=79&lineStartColumn=27&lineEndColumn=50&lineStyle=plain&_a=contents